src/Cronofy.php

```diff
@@ -785,9 +806,39 @@ public function realTimeScheduling($params)
             "event" => $params["event"],
             "availability" => $params["availability"],
             "target_calendars" => $params["target_calendars"],
-            "tzid" => $params["tzid"],
+            "tzid" => $params["tzid"]
         ];
 
+        if (isset($params['minimum_notice'])) {
+            $postFields["minimum_notice"] = $params["minimum_notice"];
+        }
+
+        if (isset($params['event_creation'])) {
+            $postFields["event_creation"] = $params["event_creation"];
+        }
+
+        if (!empty($params["formatting"])) {
+            $postFields["formatting"] = $params["formatting"];
+        }
+
+        if (!empty($params["redirect_urls"])) {
+            $postFields["redirect_urls"] = $params["redirect_urls"];
+        }
+
+        if (isset($params['callback_url'])) {
+            $postFields["callback_urls"] = array("completed_url" => $params["callback_url"]);
+        }
+
+        if (!empty($params["callback_urls"])) {
+            $postFields["callback_urls"] = $params["callback_urls"];
+
+            # If the deprecated callback_url was passed and the newer callback_urls.complete_url was not passed, modernize the payload.
+            if (!empty($params["callback_urls"]) && empty($params["callback_urls"]["completed_url"]) && (!empty($params["callback_url"]))) {
+                $callbackArray =
+                    $postFields["callback_urls"] = array_merge_recursive(array("completed_url" => $params["callback_url"]), $postFields["callback_urls"]);
+            }
```

Comment:
```suggestion
            if (empty($params["callback_urls"]["completed_url"]) && !empty($params["callback_url"])) {
                $postFields["callback_urls"]["completed_url"] = $params["callback_url"];
            }
```
Could we simplify like this? Use `callback_url` if `completed_url` isn't provided.